projects/core/src/cart/facade/cart-data.service.ts

```diff
@@ -1,32 +1,41 @@
 import { Injectable } from '@angular/core';
+import { select, Store } from '@ngrx/store';
+import { filter } from 'rxjs/operators';
+import { AuthService } from '../../auth/facade/auth.service';
 import { Cart } from '../../model/cart.model';
+import { StateWithCart } from '../store/cart-state';
+import * as fromSelector from '../store/selectors/index';
 
 export const ANONYMOUS_USERID = 'anonymous';
 
 @Injectable()
 export class CartDataService {
   private _userId = ANONYMOUS_USERID;
   private _cart: Cart;
-  private _getDetails = false;
 
-  constructor() {}
+  constructor(
```

Comment:
We both agree, that this service needs refactor. So could you hide it from the public api of the core, not to introduce possible breaking changes later?